osi_landmark.proto

```diff
@@ -927,14 +932,17 @@ message RoadMarking
         // Marking with blue color.
         //
         COLOR_BLUE = 5;
-        
+
         // Marking with red color.
+        //
         COLOR_RED = 6;
-        
+
         // Marking with green color.
+        //
         COLOR_GREEN = 7;
-        
+
         // Marking with violet color.
+        //
```

Comment:
Suggestion: the same typo might be fixed in the [line 58](https://github.com/OpenSimulationInterface/open-simulation-interface/blob/0b2c5424b080778f481a7ceec72929351d2d10cc/osi_landmark.proto#L58)